rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -2196,6 +2206,7 @@ namespace rsx
 
 	void thread::flip(int buffer)
 	{
+		this->get()->get_cycles();
```

Comment:
Why is this being called every frame? That's sure to incur an overhead

Reply:
Accidentally pushed debug code, sorry!